udata/tests/api/test_tags_api.py

```diff
@@ -31,7 +31,7 @@ def test_suggest_tags_api(self, api):
 
         for suggestion in response.json:
             assert 'text' in suggestion
-            assert suggestion['text'].startswith('test')
+            assert 'tes' in suggestion['text']
```

Comment:
`startswith` is used in other suggest test. The same issue could happen there as well?
Ex: test_suggest_datasets_api, test_suggest_organizations_api...